### PR TITLE
Setup oracle account, Add CLI commands to report oracle values

### PR DESCRIPTION
**Pull request by yerdua**
### Description

This adds a new account type that can have its private key and account address derived from the mnemonic. This price oracle account gets funded in the genesis block to cover the fees of its transactions.

The other component of this PR is two new CLI commands. One of them reports the exchange rate between a specified token (currently just `StableToken`) and `CeloGold`. The other gets the currently reported rates. This one I added to easily check if the reporting was working, and figured it might be useful enough to keep.

### Tested

The oracle account's inclusion in the genesis block and ability to act as an oracle was tested manually by spinning up a testnet and running the added CLI commands. Report transactions sent from the oracle account were successful and ones sent from other accounts were not.

### Other changes

In the code-comments documentation for `SortedOracles.report`, the `denominator` param was missing. I added an explanation of what it is.

### Related issues

- part of #981

### CODE REVIEW

File: packages/cli/src/commands/oracle/rates.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  async run() {
+    const supportedTokens: CeloToken[] = [CeloContract.StableToken]
+    const res = this.parse(GetRates)
+    if (!supportedTokens.includes(res.args.token)) {
+      return failWith(`${res.args.token} is not in the set of supported tokens: ${supportedTokens}`)
+    }
+    const sortedOracles = await this.kit.contracts.getSortedOracles()
+
+    const rates = await sortedOracles.getRates(res.args.token)
+    console.log(
```
Comment (nambrot) [inline packages/cli/src/commands/oracle/rates.ts]: Optional: You might want to use `cli.table` that we use in other cli commands as well

File: packages/cli/src/commands/oracle/report.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    const sortedOracles = await this.kit.contracts.getSortedOracles()
+    let numerator = new BigNumber(res.flags.numerator)
+    let denominator = new BigNumber(res.flags.denominator || 1)
+    if (numerator.decimalPlaces() > 0) {
+      const multiplier = new BigNumber(10).pow(numerator.decimalPlaces()).toNumber()
+      numerator = numerator.multipliedBy(multiplier)
+      denominator = denominator.multipliedBy(multiplier)
+    }
+    // const numerator = price.multipliedBy(denominator).toNumber()
```
Comment (nambrot) [inline packages/cli/src/commands/oracle/report.ts]: Remove

File: packages/cli/src/commands/oracle/report.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  static example = [
+    'report --token StableToken --numerator 1.02 --from 0x8c349AAc7065a35B7166f2659d6C35D75A3893C1',
+    'report --token StableToken --numerator 102 --denominator 100 --from 0x8c349AAc7065a35B7166f2659d6C35D75A3893C1',
+  ]
+
+  async run() {
+    const res = this.parse(ReportPrice)
+    let token: CeloToken
+    if (res.flags.token === CeloContract.StableToken) {
+      token = CeloContract.StableToken
```
Comment (nambrot) [inline packages/cli/src/commands/oracle/report.ts]: Does token have to be set if `res.flags.token` is equal to StableToken? Alternatively, you could set the `options` option on the flag to require token to be member of a set

File: packages/cli/src/commands/oracle/rates.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  static args = [{ name: 'token', required: true, description: 'Token to get the rates for' }]
+
+  static example = ['rates StableToken']
+
+  async run() {
+    const supportedTokens: CeloToken[] = [CeloContract.StableToken]
+    const res = this.parse(GetRates)
+    if (!supportedTokens.includes(res.args.token)) {
```
Comment (nambrot) [inline packages/cli/src/commands/oracle/rates.ts]: You can use the `options` option of the argument to ensure that token is in a specific set

File: packages/cli/src/commands/oracle/rates.ts (hunk 1 above)
Comment (yerdua) [reply packages/cli/src/commands/oracle/rates.ts]: oh cool! I didn't know about it until now and it's great!

**Comment by yerdua** [comment]
I split this up into multiple PRs

**Pull request closed without merging**